bam_paired.cwl

```diff
@@ -45,6 +45,7 @@ outputs:
       - clean_tables/clean_metrics
     type: File
 steps:
+# capture the version numbers of input files on Synapse
```

Comment:
All of these comments are good. The CWL specification for workflow step has a "doc" parameter that is "A long, human-readable description of this process object." Ideally, convert these comments into doc values.